src/api/appsettings.Development.json

```diff
@@ -6,7 +6,7 @@
     }
   },
   "ConnectionStrings": {
-    "BdmsContext": "Host=localhost;Username=SPAWNPLOW;Password=YELLOWSPATULA;Database=bdms"
+    "BdmsContext": "Host=localhost;Username=SPAWNPLOW;Password=YELLOWSPATULA;Database=bdms;CommandTimeout=0;"
```

Comment:
Q: Wofür brauchts das Setting `CommandTimeout`? Ist der TestDataSeed langsamer geworden?

Reply:
Nvm. Bis gestern dachte ich, dass wir durch die Einführung des Proxy von der Auth das API nur noch aus VS starten können. Dies endete fast immer in einem Timeout beim seeden. Nun läuft es jedoch wieder mit Docker.